File: androidDemo/src/main/java/com/getyourmap/androidDemo/DemoApp.java
package com.getyourmap.androidDemo;

import android.support.multidex.MultiDexApplication;
import com.getyourmap.glmap.GLMapManager;

/** Created by destman on 10/18/17. */
public class DemoApp extends MultiDexApplication {
  @Override
  public void onCreate() {
    super.onCreate();
    if (!GLMapManager.Initialize(this, this.getString(R.string.api_key), null)) { // Uncomment and insert your API key into api_key in res/values/strings.xml
      // Error caching resources. Check free space for world database (~25MB)
    }
  }
}

File: kotlinDemo/src/main/java/globus/demo/DemoApp.kt
package globus.demo

import android.app.Application
import android.util.Log
import globus.glmap.GLMapManager

class DemoApp : Application() {
    override fun onCreate() {
        super.onCreate()
        // Get an API key at https://user.globus.software/apps/ and replace api_key in strings.xml.
        if (!GLMapManager.Initialize(this, getString(R.string.api_key), null)) {
            Log.e("GLMapDemo", "GLMap initialization failed. Check free storage space and that the storage directory is writable.")
            return
        }
        GLMapManager.SetTileDownloadingAllowed(true)
    }
}
